app/src/main/java/space/taran/arkretouch/presentation/drawing/EditManager.kt

```diff
@@ -256,6 +278,8 @@ class EditManager {
                 undoRotate()
             if (undoTask == DRAW)
```

Comment:
Drawing can also be extracted into `DrawOperation` class implementing `Operation` interface.

Reply:
Exactly